src/util.js

```diff
@@ -430,6 +430,14 @@ var _ = $.extend(Mavo, {
 		return element.getAttributeNames().filter(name => regex.test(name));
 	},
 
+	// Fixes the case of attributes that are not all lowercase
+	// Especially useful for SVG attributes
+	// https://html.spec.whatwg.org/multipage/parsing.html#adjust-svg-attributes
+	getProperCasing: (element, attribute, root = "svg") => {
+		let doc = new DOMParser().parseFromString(`<${root}><${element} ${attribute}=""></${element}></${root}>`, "text/html");
```

Comment:
I think parsing is a pretty intensive operation, and these are results that don't change, so we should probably cache them in a structure. 

Then, you can cache the result in a structure like:

```js
{
	"svg": {
		"viewbox": "viewBox", // all-lowercase to proper casing
		...
	},
	"math": {...}
```

Reply:
Done! Thank you.